target/linux/ramips/dts/RT-AC54U.dts

```diff
@@ -0,0 +1,152 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rt-ac54u", "ralink,mt7620a-soc";
+	model = "Asus RT-AC54U";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "rt-ac54u:blue:power";
+			gpios = <&gpio0 9 GPIO_ACTIVE_LOW>;
+		};
+
+		usb {
+			label = "rt-ac54u:blue:usb";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&ohci_port1>, <&ehci_port1>;
+			linux,default-trigger = "usbport";
+		};
+
+		wifi {
+			label = "rt-ac54u:blue:wifi";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	gpio_export {
```

Comment:
gpio-export has been deprecated in favor of the userspace gpio-switch (see target/linux/ramips/base-files/etc/board.d/03_gpio_switches ) solution which comes with uci integration.

(However, you could also just use gpio-hog to force this on and let the user disable the leds manually. your choice).

Reply:
RT-AC51U DTS

		gpio_export {
			compatible = "gpio-export";
			#size-cells = <0>;

			enable-leds {
				gpio-export,name = "enable-leds";
				gpio-export,output = <1>;
				gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
			};
		};
	};


RT-AC51U DTS

		gpio_export {
			compatible = "gpio-export";
			#size-cells = <0>;
	
			enable-leds {
				gpio-export,name = "enable-leds";
				gpio-export,output = <0>;
				gpios = <&gpio0 10 GPIO_ACTIVE_HIGH>;
			};
		};
	};`

Reply:
@p055877632 Please see https://github.com/openwrt/openwrt/pull/1366#issuecomment-424834448 for details